README.md

```diff
@@ -1429,6 +1430,77 @@ $ m . /example
     packages.nix  sources.json  sources.nix
 ```
 
+### asBashArray
+
+Transforms a list of arguments
+to a [Bash][BASH] array.
+It can be used for passings
+several arguments from [Nix][NIX]
+to [Bash][BASH].
+
+Inputs:
+
+- args (`listOf str`):
+  list of arguments
+  to transform.
+
+Examples:
+
+1. Using it with [makeScript](#makescript):
+
+    ```nix
+    # /path/to/my/project/makes/example/main.nix
+    { makeScript
+    , ...
+    }:
+    makeScript {
+      replace = {
+        __argTargets__ = asBashArray [ "first" "second" "third" ];
+      };
+      entrypoint = ''
+        export targets=__argTargets__
+        for target in "${targets[@]}"; do
```

Comment:
this code won't run because:

```nix
''
  ${var}
''
```

is asking for the Nix's var `var` instead of the bash' var `var`